applications/openshift/api-server/api_server_api_priority_flowschema_catch_all/rule.yml

```diff
@@ -0,0 +1,51 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'Ensure catch-all FlowSchema object for API Priority and Fairness Exists'
+
+description: |-
+    Using <tt>APIPriorityAndFairness</tt> feature provides a fine-grained way
+    to control the behaviour of the Kubernetes API server in an overload
+    situation. The well-known FlowSchema <tt>catch-all</tt> should be available
+    to make sure that every request gets some kind of classification. By default,
+    the <tt>catch-all</tt> priority level only allows one concurrency share and
+    does not queue requests
```

Comment:
Can you add the `oc get flowschema` commands here?

Reply:
done